src/pyfmi/fmi.pyx

```diff
@@ -4004,25 +4108,32 @@ cdef class FMUModelBase2(ModelBase):
 
         #Load information from model
         if isinstance(self,FMUModelME2):
-            self._modelId           = decode(FMIL.fmi2_import_get_model_identifier_ME(self._fmu))
+            self._modelid           = decode(FMIL.fmi2_import_get_model_identifier_ME(self._fmu))
```

Comment:
Why change to lower case id? We usually use a capital letter to indicate a new name/word. I.e. here we have "model" and "id" -> modelId.

Reply:
I agree, an error on my side, I've fixed it